docs-site/website/versioned_docs/version-2.0.0/onnx.md

```diff
@@ -1,7 +1,6 @@
 ---
-id: version-2.0.0-onnx
+id: onnx
```

Comment:
I think we do not neet to duplicate the documentation into versioned_docs/ as this documentation will be available in v2.1. The code is now in singa:dev.

Reply:
@FeynmanDNA do you have any idea about this?